presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -357,6 +358,7 @@ public void testSingleLevelSchemaArrayOfMaps()
     }
 
     @Test
+    @Flaky(issue = "https://github.com/prestosql/presto/issues/4984", match = ".*maxCapacityHint can't be less than initialSlabSize.*")
```

Comment:
Maybe add a comment this comes from 
```
at org.apache.hadoop.hive.ql.io.parquet.write.ParquetRecordWriterWrapper
```
and this is test prep part, not the code being tested

(i guess this can be expressed in the `match` too, but didn't check)

Reply:
should we add the `@Flaky` to all tests then?